kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts

```diff
@@ -111,6 +111,12 @@ export const Preview = styled.span`
   cursor: pointer;
 `;
 
+export const ReplicaCell = styled.span.attrs({ 'aria-label': 'replica-info' })<{
```

Comment:
Why are we adding ReplicaCell to common component? It must leave in topics section

Reply:
Moved to Topics